tests/db/test_bigchain_api.py

```diff
@@ -981,36 +981,36 @@ def test_get_spent_multiple_owners(self, b, user_sk, user_pk, alice):
 
 
 def test_get_owned_ids_calls_get_outputs_filtered():
-    from bigchaindb.core import Bigchain
+    from bigchaindb.tendermint import BigchainDB
     with patch('bigchaindb.core.Bigchain.get_outputs_filtered') as gof:
```

Comment:
I think this needs to be changed too

Reply:
Yup. Good thing this test is deactivated! lol